ansible/roles/docker/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
 - apt: name=apparmor
   tags: 
     - build_ami
-- apt: name=docker-engine=1.7.0-0~trusty
+- apt: name=docker-engine=1.9.1-0~trusty
```

Comment:
We've ran into issues w/ 1.9 - honestly anything past 1.7 has led to issues with the logging infrastructure.  We're waiting for 1.11, which willhopefully fix some of those issues with the new logging drivers, and then we'll be releasing a new logging infrastructure along with it.